src/i18n/en.js

```diff
@@ -391,14 +351,8 @@ export default {
     title: `Request an update of your Plan`,
   },
   validation_messages: {
-    error_account_is_blocked_invalid_pass_HTML: `
-      For security reasons we've temporarily disabled your account.
-      <a href="${mailtoSupport}">Contact us</a>.
-      `,
-    error_account_is_canceled_HTML: `
-      Your account has been cancelled. To know more please
-      <a href="${mailtoSupport}">contact us</a>.
-      `,
+    error_account_is_blocked_invalid_pass_HTML_MD: `For security reasons we've temporarily disabled your account. [Contact us](${mailtoSupport}).`,
+    error_account_is_canceled_HTML_MD: `Your account has been cancelled. To know more please [contact us](${mailtoSupport}).`,
```

Comment:
HTML suffix is not necessary anymore, right?

```suggestion
    error_account_is_blocked_invalid_pass_MD: `For security reasons we've temporarily disabled your account. [Contact us](${mailtoSupport}).`,
    error_account_is_canceled_MD: `Your account has been cancelled. To know more please [contact us](${mailtoSupport}).`,
```

Reply:
yes you right! i removed now